### DISCUSSION TITLE
import xml - instantiate programmatically

### DISCUSSION THREAD

**Discussion by 10062244** [FreeOpcUa/opcua-asyncio#988] [Q&A] [answered]
Hello,
Is there a way to import an ObjectType via server.importxml and then instantiate an object in the namespace programmatically that references that ObjectType? If so, what would that code look like? 

I'm trying to build a server in a mold temp control unit using the Euromap 82.1 spec and the TCD_InterfaceType ObjectType. However, I don't understand how to import their xml file and then actually update the vars in the object once the xml has been imported.

Thanks

**Comment by AndreasHeine** [comment, marked as answer]
sure! 
maybe have a look at: https://github.com/umati/Sample-Server-asyncio/blob/develop/src/server.py
